atoman/gui/dialogs/infoDialogs.py

```diff
@@ -679,8 +682,10 @@ def __init__(self, pipelinePage, atomIndex, scalarsDict, vectorsDict, filterList
         self.filterList = filterList
         self.neighbourInfoWindow = None
         
-        lattice = self.pipelinePage.inputState
+        self.PBC = [1,1,1]
```

Comment:
I think the periodic boundaries need to be on for the shift atoms function to wrap atoms through the boundaries.  Some users may not want this, so it should be OK to set this to `lattice.PBC`
Is this value set by the checkboxes in the gui?

Its probably easiest if you do this, i'm not sure if I can edit the pull request now.